istio/istio.spec

```diff
@@ -376,6 +376,8 @@ mkdir -p ISTIO/src/istio.io/istio
 tar zxf %{SOURCE0} -C ISTIO/src/istio.io/istio --strip=1
 
 cp %{SOURCE1} ISTIO/src/istio.io/istio/.istiorc.mk
+
+sed -i "s/istio.io\/pkg\/version.buildVersion=.*/istio.io\/pkg\/version.buildVersion="%{version}"/" %{SOURCE2}
```

Comment:
This is not necessary. The changes in `buildinfo` file are enough, and changing the version number it is part (or should be) of the building process.

Reply:
@jwendell This updates buildinfo on the build so the user isn't expected to manually update it. We missed that in 1.1, so this fixes it.

Reply:
This is the fix for maistra-1245, not 1244.

Reply:
+1 the definitive version/release is defined within the spec file so it should be updating buildinfo as part of the build process.